app/server.go

```diff
@@ -75,6 +75,9 @@ func main() {
 		exitMtx.Lock()
 		println() // make it look nice after the ^C
 		logger.Info("[glutton ] shutting down...")
+
+		// TODO
+		// Close connections on shutdown.
```

Comment:
This is not fixed. I think we need to pass context to all routines. On interrupt context will close connections. So we will call freki shut down after glutton shutdown. I will make a PR for this.